index.html

```diff
@@ -1,11 +1,168 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>HTML Homework 3</title>
-</head>
-<body>
-
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="./node_modules/modern-normalize/modern-normalize.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+
+    <title>WebStudio</title>
+  </head>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <nav class="nav">
+            <a class="logo" href="">
+              <span class="logo__web">Web</span
+              ><span class="logo__studio studio--black">Studio</span>
+            </a>
+            <ul class="nav__list">
+              <li class="nav__item"><a href="" class="nav__link">Студія</a></li>
+              <li class="nav__item">
+                <a href="" class="nav__link">Портфоліо</a>
```

Comment:
`portfolio` link should lead us to the `Portfolio page`
```suggestion
                <a href="./portfolio.html" class="nav__link">Портфоліо</a>
```